mappings/net/minecraft/util/UserCache.mapping

```diff
@@ -21,8 +24,22 @@ CLASS net/minecraft/class_3312 net/minecraft/util/UserCache
 	METHOD method_14515 findByName (Ljava/lang/String;)Lcom/mojang/authlib/GameProfile;
 		ARG 1 name
 	METHOD method_14516 getLastAccessedEntries (I)Ljava/util/stream/Stream;
+		ARG 1 limit
 	METHOD method_14517 load ()Ljava/util/List;
 	METHOD method_14518 save ()V
+	METHOD method_30164 add (Lnet/minecraft/class_3312$class_3313;)V
+		ARG 1 entry
+	METHOD method_30165 serialize (Lnet/minecraft/class_3312$class_3313;Ljava/text/DateFormat;)Lcom/google/gson/JsonElement;
```

Comment:
We use toJson/fromJson for such static methods; see https://github.com/FabricMC/yarn/blob/21w18a/CONVENTIONS.md#serialization